components/RatingTable.vue

```diff
@@ -137,12 +143,15 @@ const isModalOpen = ref(false);
                 {{ userRating.deaths }}
               </td>
               <td class="px-6 py-4 bg-gray-50 dark:bg-gray-800">
-                {{ userRating.foodEaten }}
+                {{ (userRating.kills / userRating.deaths || userRating.kills).toFixed(2) }}
```

Comment:
Отображать "1" для соотношения 100/0 не очень правильно. Давай, для избежания неоднозначности, будем говорить "n/a", если бот ни разу не умирал.

```suggestion
                {{ userRating.deaths ? (userRating.kills / userRating.deaths).toFixed(2) : "n/a" }}
```